### ISSUE TITLE
xprop -root _DT_SAVE_MODE: AttributeError: 'ArgsParser' object has no attribute 'theme'                                                                                                                        

### ISSUE THREAD

**Issue by edouard-lopez** [bil-elmoussaoui/Hardcode-Tray#339]
**related:** #323 

### Specifications
- Hardcode-Tray version : `v3.7` (apt show `3.5.2.1~742~ubuntu16.04.1` but I run  `hardcode-tray --update-git`)
- Distribution : 

    ```
    lsb_release -a
    No LSB modules are available.
    Distributor ID: LinuxMint
    Description:    Linux Mint 18 Sarah
    Release:        18
    Codename:       sarah
    ```
- Gtk version : `2.7.4`
- Inkscape version : `0.91 r13725` if installed
- Cairo version : `` if installed
- Sni-qt version: `0.2.7+20161212~10~ubuntu16.04.1` if installed

## Expected behavior and actual behavior

### Expected 

No idea, it's my first use.

### Actual

```
hardcode-tray --conversion-tool Inkscape                                                                                                                                                                                                   [sudo] password for ed8:                                                                                                                                                                                                                     ls: cannot access 'xprop -root _DT_SAVE_MODE': No such file or directory
Traceback (most recent call last):
File "script.py", line 81, in <module>
args = ArgsParser(args, config)
File "/opt/Hardcode-Tray/modules/parser.py", line 200, in __init__
self._parse()
  File "/opt/Hardcode-Tray/modules/parser.py", line 204, in _parse
    self._parse_theme()
  File "/opt/Hardcode-Tray/modules/parser.py", line 241, in _parse_theme
    if not self.theme:
AttributeError: 'ArgsParser' object has no attribute 'theme'
```

### Steps to reproduce the problem

1. [install via PPA](https://github.com/PapirusDevelopmentTeam/papirus-icon-theme#ubuntu-and-derivatives) as describe on PapirusDevelopmentTeam/papirus-icon-theme readme

       sudo add-apt-repository ppa:papirus/papirus
       sudo apt-get update
       sudo apt-get install papirus-icon-theme
2. run

       hardcode-tray --conversion-tool Inkscape

Note: [Updating](https://github.com/bil-elmoussaoui/Hardcode-Tray/issues/323#issuecomment-283810911) didn't change the behavior.

**Comment by redpop1** [comment]
bug confirmed, same ubuntu version.

**Comment by bil-elmoussaoui** [comment]
Let me install a virtual machine and fix this! should be on master by tomorrow evening!

**Comment by orschiro** [comment]
I have the same problem on Ubuntu 17.04.

I followed the instructions and installed it from the PPA. 

Upon first run, I receive the following error:

```
orschiro@x230:~$ hardcode-tray 
[sudo] password for orschiro: 
ls: cannot access 'xprop -root _DT_SAVE_MODE': No such file or directory
Traceback (most recent call last):
  File "script.py", line 81, in <module>
    args = ArgsParser(args, config)
  File "/opt/Hardcode-Tray/modules/parser.py", line 200, in __init__
    self._parse()
  File "/opt/Hardcode-Tray/modules/parser.py", line 204, in _parse
    self._parse_theme()
  File "/opt/Hardcode-Tray/modules/parser.py", line 241, in _parse_theme
    if not self.theme:
AttributeError: 'ArgsParser' object has no attribute 'theme'
```

**Comment by bil-elmoussaoui** [comment]
Fixed on master. Just run `hardcode-tray --update-git` and run the script again and everything should be fine!
Thank you guys for reporting and happy customizing!

**Comment by orschiro** [comment]
Thanks for the fix! :-)

**Comment by edouard-lopez** [comment]
Fix this error, got a new one #340 :/
